frozendict/__init__.py

```diff
@@ -1,6 +1,7 @@
 import collections
-import operator
-import functools
+
+
+iteritems = getattr(dict, 'iteritems', dict.items)
```

Comment:
No, if we do that then the `iteritems` helper needs to be assigned/evaluated on each `__hash__` call.